src/Scene/NodeProcess.js

```diff
@@ -402,6 +444,8 @@ NodeProcess.prototype.processNode = function processNode(node, camera, params) {
             this.refineNodeLayers(node, camera, params);
         }
 
+        this.getContentLayers(node, camera, params);
```

Comment:
Nope. Keeping separate processing for imagery, elevation and feature is a good idea so let's stick to it.